src/code/z_sub_s.c

```diff
@@ -1023,13 +1023,13 @@ s16 SubS_ComputeTrackPointRot(s16* rot, s16 rotMax, s16 target, f32 slowness, f3
     f32 step;
     f32 prevRotStep;
 
-    step = (f32)(target - *rot) * (360.0f / (f32)0x10000);
+    step = (f32)(target - *rot) * (360.0f / 0x10000);
```

Comment:
```suggestion
    step = (f32)(target - *rot) * (180.0f / 0x8000);
```
Would this work? Would this not just be a theoretical `BINANG_TO_DEG` which we don't have, but would parallel:

`#define DEG_TO_BINANG(degrees) (s16)((degrees) * (0x8000 / 180.0f))`?

OoT has this macro though:
https://github.com/zeldaret/oot/blob/12f67e108abc2a3f77cda881225ff24d0c4f5f5e/include/z64math.h#L114

Reply:
Hmm, it probably is about time to add those macros. I'll revert this and work on another PR that adds them.